src/storm/logic/DiscountedCumulativeRewardFormula.h

```diff
@@ -0,0 +1,78 @@
+#pragma once
+
+#include "storm/logic/PathFormula.h"
+
+#include "storm/logic/TimeBound.h"
+#include "storm/logic/TimeBoundType.h"
+
+namespace storm {
+namespace logic {
+class DiscountedCumulativeRewardFormula : public PathFormula {
```

Comment:
Would the amount of code duplication shrink if you instead use another base class?

Reply:
Yes, that makes things a lot more compact. Thanks for the hint